src/main/protocol.ts

```diff
@@ -92,6 +92,13 @@ export const listenForProtocolHandler = () => {
       handlePotentialProtocolLaunch(url);
     }
   });
+
+  app.removeAllListeners('second-instance');
```

Comment:
Why do we need to remove all listeners here?

Reply:
I actually have no idea and just followed "tradition". Maybe testing? That'd make sense to me.

Reply:
"Testing" seems about right! I think we were registering _a lot_ of event listeners during tests and getting warnings from Node https://github.com/electron/fiddle/pull/329

Reply:
Agree, that's my understanding too about why we remove all listeners